wirepas_mqtt_library/wirepas_otap_helper.py

```diff
@@ -214,9 +214,9 @@ def get_target_scratchpad_seq_list(self):
 
         try:
             return set([i["target_scratchpad_and_action"]['target_sequence'] for i in self._status])
-        except KeyError:
+        except (KeyError, TypeError):
             # Can happen if gateway or sink doesn't support target scratchpad
-            logging.error("Cannot determine target scratchpad sequences")
+            logging.error("Cannot determine target scratchpad sequences (are sinks and gateways supporting it?")
```

Comment:
Missing close paren ')' in the printed text.

The wording could be improved, e.g. "(is scratchpad valid for the sink?)" or some such.

Reply:
I will improve the wording. In fact the message is not clear.
The problem is not from the scratchpad itself but if you use the lib on a GW < 1.4.0 or sink <5.1.0.0